addons/fatigue/initSettings.sqf

```diff
@@ -0,0 +1,18 @@
+[
+    QGVAR(highJogCoefEnabled),
+    "CHECKBOX",
+    [LSTRING(High_Jog_Coef_Enabled), LSTRING(High_Jog_Coef_Enabled_Description)],
+    LSTRING(DisplayName),
+    false,
+    1,
+    {[_this] call FUNC(toggleHighJogFactor)}
+] call CBA_fnc_addSetting;
+
+[
+    QGVAR(highJogCoef),
+    "SLIDER",
+    [LSTRING(High_Jog_Coef), LSTRING(High_Jog_Coef_Description)],
+    LSTRING(DisplayName),
+    [0, 1, 0.95, 2],
```

Comment:
I don't know what default value should we use, I just took this from your script. Any ideas?

Reply:
Idk, needs testing. This was working ok for me when I tested it with our default settings.